### DISCUSSION TITLE
Patreon

### DISCUSSION THREAD

**Discussion by MauganRa** [fabrizioschiavi/pragmatapro#197] [Ideas]
@fabrizioschiavi do you have a patreon account?  

I'm sure a lot of people would be willing to support at a monthly level for the Regular Essential version with a desktop use only (non-print) license.  

imgui's creator gets +$1250/mo:  
https://www.patreon.com/imgui/overview

**Comment by fabrizioschiavi** [comment]
Thanks for the suggestion but I don't understand very well how can Patreon work for me

**Comment by MauganRa** [comment]
It's similar to websites like 'gofundme', but it was created specifically so people could support artists.

If you create an account on Patreon, I'll create a recurring payment to you of $5/mo.  Just add a link to your patreon account to the README in the repository.  

Creating an account:
https://support.patreon.com/hc/en-us/articles/115002958403-How-do-I-become-a-creator-and-make-a-page-on-Patreon-

Set it up so you can receive monthly donations:
https://support.patreon.com/hc/en-us/articles/115004029783-How-do-I-set-up-my-payout-information-
